installer/windows/Product.wxs

```diff
@@ -79,7 +79,12 @@
     </ComponentGroup>
     
     <Property Id="WIXUI_INSTALLDIR">INSTALLFOLDER</Property>
-    <Property Id="STORJ_IDENTITYDIR">IDENTITYDIR</Property>
```

Comment:
We need this link between the `IDENTITYDIR` Directory object and the `STORJ_IDENTITYDIR` Property that is passed to the IdentityDirDlg. By removing it you may have noticed that the dialog generates an error when used to change the directory.